src/monster.cpp

```diff
@@ -4099,3 +4099,18 @@ std::function<bool( const tripoint_bub_ms & )> monster::get_path_avoid() const
         return false;
     };
 }
+
+std::vector<std::pair<std::string, std::string>> monster::get_overlay_ids() const
+{
+    std::vector<std::pair<std::string, std::string>> rval;
+
+    // get effects
+    // at this moment we share id for effect overlay for character and for monster
+    if( show_creature_overlay_icons ) {
+        for( const auto &eff_pr : *effects ) {
+            rval.emplace_back( "effect_" + eff_pr.first.str(), "" );
```

Comment:
Why do we store two strings if the second string is always empty?

Reply:
same as before, it was copied from Character::get_overlay_ids(), and second string respond for a variant of mutation/clothes; something effects do not have, but i decided to preserve it for overload compatibility